h/static/styles/vendor/icomoon.css

```diff
@@ -1,13 +1,13 @@
5CJiVPKleQOh4tEBAQEA8QLR0eQyUmTikpTiUmRB49PAADAFUAAAOrA1UAFAAjAEAAAAEiDgIVFB4CMzI+AjU0LgIjAy4DNTQ2NxcVFBYzFSUuASsBNTQmIyE1MzI2PQEzMjY9AR4DFRQGBwIAWJx0Q0N0nFhYnHRDQ3ScWCs/bVAuBAXMMiMBJwksHCsZEv8AVhEZViMyL045HzApA1VDc5xYWZt0Q0N0m1lYnHND/QMIO1p0QhMnEswrIzJTbBoigBIZVRkSVTIjEhM/UmM1Q3YuAAABAIAABAOAA1YANAAAJSIGByU+ATU0JiclHgEzMjY1NCYjIgYVFBYXBS4BIyIGFRQWMzI2NwUOARUUFjMyNjU0JiMDABgrEf7QAgICAgEtES0ZNUtLNTVLAgL+0xEtGTVLSzUZLREBMAIBSDQ0SEg0/BEPsQcPCAcPCK8QEks1NUtLNQgPB7AQE0s1NUsSELEHDgc0SEg0NEgAAAACANUAKwMrAysACgATAAAlFBYzITI2NREhEQEjJyMHIxUhNQEAMiMBViMy/gACK5Yq1iqWAlaAIzIyIwIA/gACgCsrVVUAAAABANUBgAMrAdUABAAAASE1IRUDK/2qAlYBgFVVAAAAAAIAgAArA3MDHgAFABAAADcVMwEnAQE2NC8BJiIPARc3gKAB2KD+KALzDQ1jDSMNTqBOy6AB2KD+KAGzDSMMZA0NTqBOAAAAAAIAgAArA3MDHgAFABAAADcVMwEnAQE2NC8BJiIPARc3gKAB2KD+KALzDQ1jDSMNTqBOy6AB2KD+KAGzDSMMZA0NTqBOAAAAAAMAVQDVA6sCgAAWABsAMgAAEzQ2OwE1IyIOAhUUHgI7ATUjIiY1FyE1IRUBIxUzMhYVFAYrARUzMj4CNTQuAiOmTjeqqixOOiIiOk4sqqo3Tq8BVv6qAYCqqjdOTjeqqixOOiIiOk4sAas2TlEiOU4sLE46IlFONytVVQEAUU42N05RIjpOLCxOOSIAAQErASsC1QIAAAIAAAEXNwEr1dUCANXVAAAAAAIAVQBVA6sDAAAQABcAAAEhIgYVAxQWMyEyNjURNCYjFQUlNQUlFQNV/VYkMQEyJAKqJDIyJP6r/qsBVQFVAwAyI/4AJDIyJAIAIzKr1dVW1tZWAAQAAAAfBAADgAA7AE4AWwBnAAABMCoCIyIOAhUUHgIzOgE3DgEVFBYXIgYjIg4CFRQeAjMyPgI1NCYnLgE1NDY3PgE1NCYnMzcDHgEVFAYjIiY1NDYXMhYXHgEXAy4BJyY2Fx4BFxYGJyU1IxUjFTMVMzUzNQIvQVhaGS1UQScdNksuBg0GBggVEAwXDDdhRygsS2Q3QGJDIyk1EjUTISErLSpVPF4CAldsTWR2TRIhDyo5CKQ0VgkJPTQzVgkJPDQCE0DAwEDAA4AgNkcnKUc1HgEMGQ0XJQ4BITVGJSQ6KRYhNkYkOk4mDTQSFh4ZGkcrMlgRK/1xBw4HO1BUOzlTAQYEHiohASMCYkREXAIBYEREXgGswMBAwMBAAAAAAAEAAP/ABAADwAAkAAABISIGFREUFjMhESM1MzU0NjsBFSMiBh0BMwcjETMyNjURNCYjA1X9VkdkZEcBVYCAXkKgoA0TsCCQ1UdkZEcDwGRH/VZHZAHAgGBCXoATDWCA/kBkRwKqR2QAAQAAACAEAANgAEcAAAEOAQc+ATcOAQcuASMiDgIVFBYXLgMnDgEVFBYXLgEnMBQxFBYXDgEjIiYnHgEXDgEjIiYnHgMzMj4CNTQmNT4BNwQAHD0gITAMH0MkHFAtLEw5IQMCQXtuYCcNDzMqGjAVYUgOGw8KFAkUa0U2hUoNGQwjTFFWLJHfmE4BHzUVAv4NEQMTPCUSGgceJCE5TCwMGAwDIjlOMBg1HTdcHAEOCwJNcw4EBAICPlICKjACARYjGA1trNVoBg4HFjcgAAAAAAEAAAABgACj/yc1Xw889QALBAAAAAAA0jFYbAAAAADSMVhsAAD/qwQAA8AAAAAIAAIAAAAAAAAAAQAAA8D/wAAABAAAAAAABAAAAQAAAAAAAAAAAAAAAAAAADIEAAAAAAAAAAAAAAACAAAABAAA1QQAAJIEAACrBAAAgAQAAGQEAAArBAAAKwQAAVUEAADVBAAA1QQAAFUEAAAABAAA1QQAAIAEAAAABAABKwQAAQAEAABrBAAAVQQAANUEAAEABAAAVQQAAIAEAABVBAABbgQAAAAEAABVBAAAkgQAANUEAAGrBAAAJgQAAAgEAAAIBAAAAwQAAFUEAACABAAA1QQAANUEAACABAAAgAQAAFUEAAErBAAAVQQAAAAEAAAABAAAAAAAAAAACgAUAB4AOABKAJYA1AFgAa4CHAIwAkoCZAKYAtQC6gMIAzIDagOCA9IEFAQwBEwEkgS6BOwE/gVOBYYFmAWyBb4GGgamB6AIbgjICRYJOglKCW4JkgnYCeYKEAqgCtQLPAAAAAEAAAAyAK8ABgAAAAAAAgAAAAAAAAAAAAAAAAAAAAAAAAAOAK4AAQAAAAAAAQABAAAAAQAAAAAAAgAHACoAAQAAAAAAAwABACQAAQAAAAAABAABAD8AAQAAAAAABQALAAMAAQAAAAAABgABACcAAQAAAAAACgAaAEIAAwABBAkAAQACAAEAAwABBAkAAgAOADEAAwABBAkAAwACACUAAwABBAkABAACAEAAAwABBAkABQAWAA4AAwABBAkABgACACgAAwABBAkACgA0AFxoAGhWZXJzaW9uIDEuNQBWAGUAcgBzAGkAbwBuACAAMQAuADVoAGhoAGhSZWd1bGFyAFIAZQBnAHUAbABhAHJoAGhGb250IGdlbmVyYXRlZCBieSBJY29Nb29uLgBGAG8AbgB0ACAAZwBlAG4AZQByAGEAdABlAGQAIABiAHkAIABJAGMAbwBNAG8AbwBuAC4AAAADAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA')
```

Comment:
@tilgovi Thanks - I looked at both postcss-assets and postcss-url for this and I think we'll end up using one or the other in future.